docs/reference/tools-reference/llm-tool.md

```diff
@@ -158,3 +158,75 @@ In LLM tool, the prompt is transformed to match the [openai messages](https://pl
     }
 ]
 ```
+
+### Sample 3
+```jinja
+# system:
+You are a helpful assistant.
+
+{% for item in chat_history %}
```

Comment:
Please modify line 98 as we now support one more role 'tool' and its unique properties.

Reply:
updated, please help to take another look